### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by corpooo** [FelixKratz/SketchyBar#45] [General]
Just wanted to say thank you!

I find this project extremely awesome. I had lots of fun configuring my bar and plugins and seeing how the project grows!

I also opened a lot of issues with questions and you've always responded fast and with a current/future solution!

Thank you one more time and keep up the good work! 😄

**Comment by FelixKratz** [comment]
This just made my day!

**Reply by corpooo** [reply]
Glad you liked it!

Let us know how we can support this project. 😄

**Reply by zcag** [reply]
+1 on everything o this post, since some time has passed since i wanna reiterate, would love to have a way of supporting this project. Hopefully i'm not missing something but do we have a way to support currently?

**Reply by FelixKratz** [reply]
I have created a small section in the README for ways to support the project: https://github.com/FelixKratz/SketchyBar#supporting
